### DISCUSSION TITLE
Treesitter Question

### DISCUSSION THREAD

**Discussion by C4E5G3** [NotAShelf/nvf#439] [Q&A]
TextObjects are not builtin with Treesitter. 

I've tried creating a hybrid Treesitter config, builtin + Lua, and a pure Lua config, treating Treesitter as an imported plugin, but neither work.

Any pointers as to how to do this?

**Comment by NotAShelf** [comment]
I believe @horriblename uses TS textobjects, maybe he might be able to answer your question. 

If not, I can simply add a module - been meaning to do it for a while now anyway, and better to support it natively.
